doc-content/enterprise-only/project-data/container-alias-con.adoc

```diff
@@ -0,0 +1,43 @@
+[id='container-alias-con_{context}']
+= Container alias
```

Comment:
@hmanwani-rh , This section still feels quite rough to me for enterprise and uses some unrefined language and formatting. Too many things to flag individually, so I've pasted below a proposed rewrite of the section. If you agree, feel free to copy and paste entirely.

Some general things to watch for going forward, based on what I see here:
* Remember never to hard code "KIE Server". Use the doc variable "{KIE_SERVER}". And note that it's capitalized "KIE Server". I suggest also reviewing all of the `document-attributes...` files in the `_artifacts/` folder of our project for a refresher of the variables we should be using in docs.
* You've called it "container alias" almost like a proper noun, as in "Use container alias to" or "for information about container alias, see ...", but it's not proper. Needs to be "a container alias" or "container aliases", or in full, "a KIE container alias" on first mention, which is what it is in full. I've commented where applicable here. Just letting you know.
* Per IBM style guide, avoid passive constructs and use active, user-centered language when applicable (do this or you can do this, not this is done or can be done). Likewise "However, usage of container aliases depends on the operation that you want to perform." is quite passive and I'm not sure the active version provides much value. I've removed, imo. Also "Container aliases are typically used for ..." should be "You typically  use container aliases for ...".
* Per IBM style guide, avoid empty "it is/there is/there are". Find a concrete subject and object. (Instead of "It is possible to link ...", "You can link ...")
* Per IBM style guide (and Chicago and others), use serial commas, as in "When you update, build, and deploy" not "When you update, build and deploy".
* Be sure to use complete and clear sentences. "Consider an example, if ..." and "Consider there is ..." are incorrect or incomplete. Also "Every version contains a different `containerID`, the associated client applications need to be aware of all the versions every time when interacting with the projects." is a fragment and I'm not sure what you mean (I've rewritten as best as I've understood). 
* For bulleted lists, do not use an ending period if all listed items are incomplete sentences. If at least one is a complete sentence, then add period to all for consistency. If all are complete, then obviously all periods.
* When you use a URL or API endpoint as an example that's not literally a clickable link, escape the string with /, as it "/https...". This renders it without clickability in the output. Side note, the API URL that you listed is not "a request" as you noted, but merely the API endpont that you can send GET, POST, or whatever requests to.

Proposed text for this section:

```
= KIE container aliases

An alias for a KIE container (deployment unit) is a proxy in the {KIE_SERVER} instance that helps in handling different versions of the same container deployment. You can link a single alias to different versions of a container. When the container is upgraded, the linked alias automatically points to the new version of the container. For information about creating a KIE container alias, see xref:deployment-unit-create-proc_packaging-deploying[].

For example, if a client application is changing every time a new version of a container is deployed, then the client application can point to the container alias. When a new container version is deployed, the associated alias is updated and all the requests are routed automatically to the new container without changing the client application.

Consider an example project that contains a single process and uses the following properties:

* `GroupId`: org.jbpm
* `ArtifactId`: my-project
* `Version`: 1.0
* `containerID`: my-project

When you update, build, and deploy the above project, the associated project is updated in the {KIE_SERVER} with the latest version and contains the following properties:

* `GroupId`: org.jbpm
* `ArtifactId`: my-project
* `Version`: 2.0

If you want to deploy the latest version of the project, you need to update the `containerID` as `my-project2` because the `my-project` container points to the old version.

[NOTE]
====
Every project version contains a different `containerID` name. The associated client applications need to be aware of all  versions of the projects they interact with.
====

Container aliases also help you manage KIE containers. You can set the container aliases explicitly when creating a container, or implicitly based on the associated `ArtifactId` name. You can add a single alias to multiple containers if required. If you do not specify a container alias, then the `ArtifactId` of a project is set as the container alias by default.

When you set an alias for multiple containers that contain different `GroupId` and `ArtifactId` names, then you can use the same alias every time to interact with the {KIE_SERVER}. 

You typically use container aliases in the following use cases:

* *Starting a new process instance* on the client application with the latest version of the process
* *Interacting with an existing process* of a specific version
* *Interacting with an existing task* in a process
* *Interacting with a process definition* image and form

For example, after you deploy the 1.0 version of a project, you send a POST request to the following {KIE_SERVER} REST API endpoint to start the process in the project:

`/http://localhost:8230/kie-server/services/rest/server/containers/my-project/processes/evaluation/instances`

The sent request starts a new process instance from `org.jbpm:my-project:1.0` in which `my-project` is defined as the container alias. Later, when you deploy the 2.0 version of the project and send the same request, a new instance starts from `org.jbpm:my-project:2.0`. You can deploy the latest version of the process without adding the `containerID` name.
```

Reply:
Done, thanks @sterobin, will take care of this in future